website/docs/containers/registries/index.md

```diff
@@ -2,18 +2,252 @@
 sidebar_position: 3
 title: Registries
 description: Working with container registries
-tags: [podman-desktop, containers, registries]
+tags: [podman-desktop, containers]
 keywords: [podman desktop, podman, containers, registries]
 hide_table_of_contents: false
+image: /img/docs/containers/registries/img/registries.png
 ---
 
-# Working with container registries
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-Podman Desktop has access to public registries by default.
+# Setting up container registries
 
-You might want to configure Podman Desktop to:
 
-- [Authenticate to a preconfigured registry](/docs/containers/registries/authenticating-to-a-preconfigured-registry).
-- [Add an Insecure Registry to Podman Desktop](/docs/containers/registries/insecure-registry).
 
-![Registries page](img/registries.png)
+## Before you start
+
+Before you start, you should:
+
+- Get authentication details for your container registry:
+  - Registry URL.
+  - User name.
+  - Password, or OAuth secret.
+- Get the fully qualified name of a private image stored in your registry, such as `my-registry.tld/my-repository/my-image`.
+- Get the fully qualified image name that your registry requires to push an image, such as `my-registry.tld/my-repository/my-image`.
```

Comment:
```suggestion
```

Reply:
Commit at will.

Reply:
What would you think about the idea of putting the section into a "note"

:::note
Before you start, you should:

- Get authentication details for your container registry:
  - Registry URL.
  - User name.
  - Password, or OAuth secret.
- Get the fully qualified name of a private image stored in your registry, such as `my-registry.tld/my-repository/my-image`.
- Get the fully qualified image name that your registry requires to push an image, such as `my-registry.tld/my-repository/my-image`.
:::

Reply:
Let's try

```suggestion
:::tip[Before you start]

Before you start, you should:

- Get authentication details for your container registry:
  - Registry URL.
  - User name.
  - Password, or OAuth secret.
- Get the fully qualified name of a private image stored in your registry, such as `my-registry.tld/my-repository/my-image`.
- Get the fully qualified image name that your registry requires to push an image, such as `my-registry.tld/my-repository/my-image`.

:::
```